core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -152,6 +154,9 @@ public MetadataVersion metadataVersion() {
     public Map<Integer, Map<String, String>> perBrokerOverrideProperties() {
         return perBrokerOverrideProperties;
     }
+    public Map<String, String> displayTags() {
```

Comment:
Please update `nameTags`

Reply:
I thought that `nameTags` was for other purposes, thus separated them and use different name for the tags that I am adding. If they are the same concept, let me rename it and combine them together

Reply:
> If they are the same concept, let me rename it and combine them together

It seems to me that one "tags" method is enough. That method returns the "modified" tags ( raw + others)